apps/worker/src/app/workflow/usecases/run-job/run-job.usecase.ts

```diff
@@ -131,17 +141,6 @@ export class RunJob {
     }
   }
 
-  @CachedEntity({
-    builder: (command: { _id: string; environmentId: string }) =>
-      buildNotificationTemplateKey({
-        _environmentId: command.environmentId,
-        _id: command._id,
-      }),
-  })
```

Comment:
Caching is less effective when retrieving `notification` rather than `notificationTemplate` because the relation has changed:
* Previously: 1 `notificationTemplate` for all Subscriber `notifications`
* Now: 1 `notification` per 1 Subscriber `notification`

With that in mind, I've opted to remove caching for the `notification` retrieval as the cache is only used by steps in the same workflow. I suggest we monitor the transaction and add caching later if necessary.